keyboards/niu_mini/readme.md

```diff
@@ -13,4 +12,4 @@ Make example for this keyboard (after setting up your build environment):
 
     make niu_mini:default:dfu
```

Comment:
What do you think about:

```suggestion
    make niu_mini:default      # Compile firmware only
    make niu_mini:default:dfu  # Compile and attempt to flash
```

?

Reply:
I'd say we should copy how it is in the ps2avrgb readme template (which uses the `:flash` target), and add it to the AVR template next.

Reply:
I actually forgot `:flash` was a thing.

Reply:
Well, it's still pretty new.  :)